api/src/services/utils.ts

```diff
@@ -147,4 +147,12 @@ export class UtilsService {
 			}
 		);
 	}
+
+	async clearCache(): Promise<void> {
+		if (this.accountability?.admin !== true) {
+			throw new ForbiddenError();
+		}
```

Comment:
I think perhaps we should keep this check in the controller only so it is not required to fake an accountability object (when running from a crontab event for instance)

Reply:
Wasn't quite sure at first either, but I think the check makes more sense as part of the service, as an extension might get `accountability` from a request as well and pass it along to the service. Feels safer and also it's consistent with other service methods where the permission check takes place at that level too.

Reply:
I think it's fine to have that check here. Clearing the cache is admin only; in things like flows and operations there is additional options to "Run as Admin", so we should have things covered

Reply:
It's the same as with every other system service 👍🏻

Reply:
> [...] in things like flows and operations there is additional options to "Run as Admin", so we should have things covered

Good point 👍🏻